src/common/mipmap_cache.c

```diff
@@ -1019,6 +1019,27 @@ void dt_mipmap_cache_remove(dt_mipmap_cache_t *cache, const uint32_t imgid)
     }
   }
 }
+void dt_mipmap_cache_remove_at_size(dt_mipmap_cache_t *cache, const uint32_t imgid, dt_mipmap_size_t mip)
+{
+  // get rid of the thumbnails:
+  const uint32_t key = get_key(imgid, mip);
+  dt_cache_entry_t *entry = dt_cache_testget(&_get_cache(cache, mip)->cache, key, 'w');
+  if(entry)
+  {
+    ASAN_UNPOISON_MEMORY_REGION(entry->data, dt_mipmap_buffer_dsc_size);
```

Comment:
Is this expected or a left over the debug session?

Reply:
This new fct is just a copy/paste of the dt_mipmap_cache_remove fct for only 1 mip level. So I think it's needed...